nl_server/loader.py

```diff
@@ -45,6 +47,9 @@ def load_server_state(app: Flask):
   flask_env = os.environ.get('FLASK_ENV')
 
   embeddings_dict = _load_yaml(flask_env)
+  vertex_ai_models = {}
+  if not is_custom_dc():
```

Comment:
This is only gated by !custom-dc?  We don't want it on prod etc until we setup the backend service, etc.  So for now, only include in autopush?

Reply:
done & also included in local and tests (since I added an integration test for using a vertex ai model)